README.md

```diff
@@ -3,10 +3,6 @@ Nginx container images
 
 [![Build and push container images to quay.io registry](https://github.com/sclorg/nginx-container/actions/workflows/build-and-push.yml/badge.svg)](https://github.com/sclorg/nginx-container/actions/workflows/build-and-push.yml)
 
-nginx-container 1.18 status: [![Docker Repository on Quay](https://quay.io/repository/centos7/nginx-118-centos7/status "Docker Repository on Quay")](https://quay.io/repository/centos7/nginx-118-centos7)
```

Comment:
Could you please add info about building and publishing fedora images to the readme?

Reply:
I will look at it, but basically, If build-and-push is working and it is green, then everything, like C8S, C9S, Fedora, and CentOS 7 is working properly, but reference to the images is fine. The status will not be ok, I will change it somehow.

Reply:
Fixed by https://github.com/sclorg/nginx-container/pull/183/commits/3f7a254a89a8eae7fa81039046ab8b484a5ee5b4

Reply:
There is also line:
`This repository contains Dockerfiles for Nginx images for OpenShift. Users can choose between RHEL and CentOS based images.` which is no longer true, and
```
Versions

Nginx versions currently provided are:

    [nginx-1.18](https://github.com/sclorg/nginx-container/blob/3f7a254a89a8eae7fa81039046ab8b484a5ee5b4/1.18)
    [nginx-1.20](https://github.com/sclorg/nginx-container/blob/3f7a254a89a8eae7fa81039046ab8b484a5ee5b4/1.20)

RHEL versions currently supported are:

    RHEL7
    RHEL8

CentOS versions currently supported are:

    CentOS7
    CentOS Stream 8
    CentOS Stream 9
```
where Fedora should also be added.

Reply:
I will update the row in OpenShift use case, but among all sclorg repositories, `Fedora versions currently supported are:` is not mentioned at all.

Reply:
I would prefer to update documentation as a separate Pull Request. But it brings one thing to my mind. Do we want to review the documentation among all repositories?

Reply:
We probably should review it.